nf_core/pipelines/lint/__init__.py

```diff
@@ -52,32 +75,30 @@ class PipelineLint(nf_core.utils.Pipeline):
         warned (list): A list of tuples of the form: ``(<warned no>, <reason>)``
     """
 
-    from .actions_awsfulltest import actions_awsfulltest  # type: ignore[misc]
-    from .actions_awstest import actions_awstest  # type: ignore[misc]
-    from .actions_ci import actions_ci  # type: ignore[misc]
-    from .actions_schema_validation import (  # type: ignore[misc]
-        actions_schema_validation,
-    )
-    from .configs import base_config, modules_config  # type: ignore[misc]
-    from .files_exist import files_exist  # type: ignore[misc]
-    from .files_unchanged import files_unchanged  # type: ignore[misc]
-    from .merge_markers import merge_markers  # type: ignore[misc]
-    from .modules_json import modules_json  # type: ignore[misc]
-    from .modules_structure import modules_structure  # type: ignore[misc]
-    from .multiqc_config import multiqc_config  # type: ignore[misc]
-    from .nextflow_config import nextflow_config  # type: ignore[misc]
-    from .nfcore_yml import nfcore_yml  # type: ignore[misc]
-    from .pipeline_name_conventions import (  # type: ignore[misc]
-        pipeline_name_conventions,
-    )
-    from .pipeline_todos import pipeline_todos  # type: ignore[misc]
-    from .readme import readme  # type: ignore[misc]
-    from .schema_description import schema_description  # type: ignore[misc]
-    from .schema_lint import schema_lint  # type: ignore[misc]
-    from .schema_params import schema_params  # type: ignore[misc]
-    from .system_exit import system_exit  # type: ignore[misc]
-    from .template_strings import template_strings  # type: ignore[misc]
-    from .version_consistency import version_consistency  # type: ignore[misc]
+    # Import all linting tests as methods for this class
+    actions_awsfulltest = actions_awsfulltest
```

Comment:
what is this assignment doing?

Reply:
makes them callable in instances of the class (before it threw a pylance error)